tasks/__init__.py

```diff
@@ -467,16 +467,10 @@ def remove_failures_from_testmon(ctx, db_path=None):
 
 @task
 def travis_setup(ctx):
-    ctx.run('npm install -g bower', echo=True)
-
     with open('package.json', 'r') as fobj:
         package_json = json.load(fobj)
         ctx.run('npm install @centerforopenscience/list-of-licenses@{}'.format(package_json['dependencies']['@centerforopenscience/list-of-licenses']), echo=True)
 
-    with open('bower.json', 'r') as fobj:
-        bower_json = json.load(fobj)
-        ctx.run('bower install {}'.format(bower_json['dependencies']['styles']), echo=True)
-
```

Comment:
I remember you mentioned the bower but I forgot the reason. Is this an extra clean-up out of the scope of the ticket/PR?

Reply:
It speeds up the tests a lot and is part of the gradual process of striping away dependencies that won't be necessary in the future.